data/example.json

```diff
@@ -498,22 +498,18 @@
       "responses": [{ "value": "Yes" }]
     },
     {
-      "metadata": "",
```

Comment:
I removed this as it seemed like an issue with the example - I can't recreate this when uploading images?